### ISSUE TITLE
Cannot parse inline string array

### ISSUE THREAD

**Issue by hhsnopek** [reshape/expressions#3]
With Sugarml

```
ul: each(loop='item of ["item1", "item2"]')
  li {{ item }}

ul: each(loop="item of ['item1', 'item2']")
  li {{ item }}
```

Numeric Arrays do work as intended

Error: (Non-user friendly error)
`Module build failed: SyntaxError: Unexpected token (1:953)`

**Comment by hhsnopek** [comment]
Thus you're unable to loop thru `Object.keys({ foo: "bar", baz: "zar" })`

**Comment by jescalan** [comment]
Object = `in`, Array = `of`, same as javascript 😁 

https://github.com/reshape/expressions#loops

Although there should certainly be a better error message

**Comment by hhsnopek** [comment]
whoops, right - I do have `of` locally and it still fails - I always type it wrong 😁

**Comment by hhsnopek** [comment]
fails:

```
ul: each(loop="item of ['item1', 'item2']")
  li {{ item }}
```

passes:

```
ul: each(loop="item of [1, 2]")
  li {{ item }}
```

**Comment by jescalan** [comment]
Weird. I'll look into this shortly. Thanks for the report!

**Comment by hhsnopek** [comment]
Modifying the results for locals with strings still pass, but inline for loops with defined array's do not pass within spike

**Comment by jescalan** [comment]
Sorry I'm not sure I understand

**Comment by hhsnopek** [comment]
I modified the test locals to use `['1', '2', '3']` and they still passed

**Comment by jescalan** [comment]
Closed by https://github.com/reshape/sugarml/commit/50798447b97a318c242416faf05d9f3c14ac5ab7, shipped in `sugarml@0.2.2`.

Thanks for catching this!
